### DISCUSSION TITLE
Releasing v2.2

### DISCUSSION THREAD

**Discussion by airbreather** [NetTopologySuite/NetTopologySuite#501] [General]
From @FObermaier in https://github.com/NetTopologySuite/NetTopologySuite/pull/485#issuecomment-779268150:

> I'd like to release v2.2 soon, that is why i assigned some issues to milestone 2.2. I'd be grateful if you could have a look at those and tell me if you'd like to see any of these addressed before a new release. Thanks!

Of the items in https://github.com/NetTopologySuite/NetTopologySuite/milestone/8, I don't personally have stake in any of them, but:
1. #447 seems like it has everything that's needed to move on, now that we've done #483?
1. There's been more progress towards GML3, so maybe it's worth looking into wrapping up what's left for #341?

**Comment by FObermaier** [comment]
Are there others things you like to see resolved before a v2.2 release? 
If not, could you do that? IIRC you did all the v2 releases, so far.

**Comment by FObermaier** [comment]
@airbreather, how about releasing v2.2 now, or is there still sth. in the pipeline? JTS has released v1.18.1 already.

**Reply by airbreather** [reply]
I was going to do it after I find the time to do #320 and #348 since those are the last two on the milestone, though clearly I haven't found the time in a few weeks now.

I'll do the release soon, probably tonight.

**Reply by FObermaier** [reply]
Thanks 👍
